addons/point_of_sale/static/src/app/models/pos_order.js

```diff
@@ -95,7 +95,9 @@ export class PosOrder extends Base {
             .filter((p) => !p.is_change)
             .map((p) => p.export_for_printing());
         return {
-            orderlines: this.lines.map((l) => omit(l.getDisplayData(), "internalNote")),
+            orderlines: this.config.orderlines_sequence_in_cart_by_category
+                ? this.getSortedOrderlines().map((l) => omit(l.getDisplayData(), "internalNote"))
+                : this.lines.map((l) => omit(l.getDisplayData(), "internalNote")),
```

Comment:
```suggestion
            orderlines: lines.map((l) => omit(l.getDisplayData(), "internalNote")),
```

where `const lines = this.config.orderlines_sequence_in_cart_by_category ? this.getSortedOrderlines() : this.lines`

Reply:
Hello @caburj Changes are done please check
Thanks!